R/net.mod.infection.R

```diff
@@ -300,7 +302,7 @@ discord_edgelist <- function(dat, at) {
   if (tergmLite == TRUE) {
     el <- dat$el[[1]]
```

Comment:
input network here too (there is a parallel structure for the edgelist for multi-network simulations using tergmLite)

Reply:
Req. changes updated and passing on Travis.